### ISSUE TITLE
Cannot print index returned by audinterface.Segment

### ISSUE THREAD

**Issue by hagenw** [audeering/audinterface#119]
Tested with version 1.0.2

```python
import audb
import audinterface

def process_func(signal, sampling_rate):
    return audinterface.utils.signal_index(starts=[0.12], ends=[1.76])

db = audb.load('emodb')
interface = audinterface.Segment(process_func=process_func)
index = interface.process_index(db.files[:2])
```

But then
```python
>>> index
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/audeering.local/hwierstorf/.envs/test/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 1233, in __repr__
    data = self._format_data()
  File "/home/audeering.local/hwierstorf/.envs/test/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 1275, in _format_data
    return format_object_summary(
  File "/home/audeering.local/hwierstorf/.envs/test/lib/python3.8/site-packages/pandas/io/formats/printing.py", line 436, in format_object_summary
    word = tail_item + sep + " "
TypeError: can only concatenate tuple (not "str") to tuple
```

When I add the index to a dataframe it can be printed, so maybe this is a `pandas` error?

```python
>>> pd.DataFrame([0, 0], index=index)
                                                                                                 0
file                                              start                  end                      
/home/audeering.local/hwierstorf/audb/emodb/1.... 0 days 00:00:00.120000 0 days 00:00:01.760000  0
                                                                         0 days 00:00:01.760000  0
```

**Comment by hagenw** [comment]
I can also replicate this when I create the index myself:

```python
import audformat

files = list(db.files[:2])
starts = [0.12, 0.12]
ends = [1.76, 1.76]
index = audformat.segmented_index(files, starts, ends)
```

and then

```python
>>> index
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/audeering.local/hwierstorf/.envs/test/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 1233, in __repr__
    data = self._format_data()
  File "/home/audeering.local/hwierstorf/.envs/test/lib/python3.8/site-packages/pandas/core/indexes/base.py", line 1275, in _format_data
    return format_object_summary(
  File "/home/audeering.local/hwierstorf/.envs/test/lib/python3.8/site-packages/pandas/io/formats/printing.py", line 436, in format_object_summary
    word = tail_item + sep + " "
TypeError: can only concatenate tuple (not "str") to tuple
```

**Comment by hagenw** [comment]
Interestingly, it seems to depend on the string of the files:

```python
files = ['a', 'b']
starts = [0.12, 0.12]
ends = [1.76, 1.76]
audformat.segmented_index(files, starts, ends)
```
returns

```
MultiIndex([('a', '0 days 00:00:00.120000', '0 days 00:00:01.760000'),
            ('b', '0 days 00:00:00.120000', '0 days 00:00:01.760000')],
           names=['file', 'start', 'end'])
```

**Comment by hagenw** [comment]
This was indeed a `pandas` bug, see https://github.com/pandas-dev/pandas/issues/53537 and should be fixed with a new `pandas` release.

**Comment by hagenw** [comment]
This should be fixed in `pandas>=2.1.0`.

**Comment by hagenw** [comment]
This was indeed fixed in `pandas`.
